rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -1,8 +1,19 @@
 # This is the message to communicate a parameter's descriptor.
 
 string name
-uint8 type # Enum defined in ParameterType.msg
-# If 'true' then the value cannot change after it has been initialized
+
+# Enum values are defined in the `ParameterType.msg` message.
+uint8 type
```

Comment:
Anyone know why we decided to not place a `type` or `data` field in `ParameterType.msg` and then use `ParameterType type` here? Just to avoid `...type.data` or `...type.type`?

It would be nice to not have to imply where the constants for this field are using a comment.

Reply:
I guess we'll leave this as-is.